packages/common/container-definitions/src/error.ts

```diff
@@ -83,7 +83,18 @@ export interface ContainerWarning extends IErrorBase {
 }
 
 /**
- * Represents errors raised on container.
+ * Represents errors raised on container. Check the "See below" section for commonly thrown error types.
```

Comment:
(100% nit-pick): The generated docs might look a little nicer if you move the "Check the ..." sentence to the top of the `@see` comment block. That way it's introducing the subsequent list directly.